commit f7391d904999138158d9bca9e7c34ef18550e2fe
Author: tenzap <fabstz-it@yahoo.fr>
Date:   Sat Mar 26 11:23:32 2022 +0100

    fix compilation of TagsEdit.cpp
    
    qOverload appeared with qt5.7
    
    Reported error:
    keepassxc-2.7.0-src/src/gui/tag/TagsEdit.cpp:414:34: error: use of undeclared identifier 'qOverload'
            connect(completer.get(), qOverload<QString const&>(&QCompleter::activated), [this](QString const& text) {
                                     ^

diff --git a/src/gui/tag/TagsEdit.cpp b/src/gui/tag/TagsEdit.cpp
index b4bfb7f..a347940 100644
--- src/gui/tag/TagsEdit.cpp
+++ src/gui/tag/TagsEdit.cpp
@@ -411,7 +411,7 @@ struct TagsEdit::Impl
     void setupCompleter()
     {
         completer->setWidget(ifce);
-        connect(completer.get(), qOverload<QString const&>(&QCompleter::activated), [this](QString const& text) {
+        connect(completer.get(), static_cast<void (QCompleter::*)(QString const&)>(&QCompleter::activated), [this](QString const& text) {
             currentText(text);
         });
     }
